### DISCUSSION TITLE
src location

### DISCUSSION THREAD

**Discussion by dedicate617** [alexzhornyak/ScxmlEditor-Tutorial#59] [Q&A]
where is the source of the ScxmlEditor,tks

**Comment by alexzhornyak** [comment]
Unfortunately, the source code is not available, you can only download the app deployment (binaries) by the link on **readme** page

**Comment by dedicate617** [comment]
get it, it's a wonderful tool, but unfortunately i can't intergrated into my app

**Reply by alexzhornyak** [reply]
Source code is available
